point_pattern.py

```diff
@@ -121,6 +150,20 @@ def average_nearest_neighbor_distance(points):
     """
     mean_d = 0
 
+    shortDistanceList = []
+
+    for firstPoint in points:
+        pointInList = 500
```

Comment:
I would swap in `math.inf` here, so that you have a really big starting distance.